File: MCP_Backend-master/src/main/java/com/greybox/mediums/models/ChargeTierData.java
package com.greybox.mediums.models;

import com.greybox.mediums.entities.ServiceChargeTier;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ChargeTierData {
    private ServiceChargeTier[] data;
    private Integer chargeId;
    private String createdBy;
    private Date createDate;
}

File: MCP_Frontend-main/src/app/shared/models/charge-tier-data.ts
export class ChargeTier {
    chargeTierId: number;
    chargeId: number;
    tierNo: number;
    fromAmt: number;
    toAmt: number;
    chargeAmt: number;
    createDate: Date;
    createdBy: string;
    modifyDate: Date;
    modifiedBy: string;
}

export class ChargeTierData {
    data: ChargeTier[];
    chargeId: number;
    createdBy: String;
    createDate: Date;
}